examples/v1beta1/pipelineruns/pipelinerun.yaml

```diff
@@ -90,7 +90,7 @@ spec:
       runAsUser: 0
   - name: write-digest
     workingDir: $(workspaces.source.path)
-    image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/imagedigestexporter:latest
+    image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/imagedigestexporter:v0.11.1
```

Comment:
Can this example be updated to not use the imagedigestexporter image? I believe we mentioned in the WG yesterday that there's a suitable kaniko flag that can replace it.

Reply:
yeah we should just replace this kaniko task with the latest version of kaniko from the Hub

Reply:
> yeah we should just replace this kaniko task with the latest version of kaniko from the Hub
For v1beta1 I think we can but resolvers are behind beta feature flag in v1.

Reply:
Thanks for opening the issue! even if we can't use remote resolution here we should still migrate off usage of this image though

Reply:
Following up on the migration off the dependencies of the imageDigestExporter image, @QuanZhang-William has helped me with the investigation of errors with the latest knaiko task at https://github.com/tektoncd/pipeline/issues/5727#issuecomment-1306169832. According to the findings that the cataglog latest kaniko task works fine, it is the old `url` provided at https://github.com/GoogleContainerTools/skaffold/blob/main/examples/microservices/leeroy-web/Dockerfile was not compatible with the latest kaniko task. So here are two options moving forward to unblock this:

- [ ] delete the population of the image digest if the purpose of the `pipelinerun.yaml` was not to deploy the microservice from skaffold
- [ ] not to use the skaffold images examples for params, which would change the microservices to be built in `pipelinerun.yaml` also

Reply:
Do we have any preference on maintaining the existent pipelinerun.yaml examples?

Reply:
It looks to me that the goal of the example is to showcase a typical CICD pipeline (i.e. clone -> test -> build & push -> deploy). If that is the case I think we should go for #2.

Reply:
`pipelinerun.yaml` is one of the original examples and has been a great resource showcasing a basic CI/CD use case, please preserve it if possible, thanks!